src/lib.rs

```diff
@@ -1016,11 +1031,14 @@ impl Triangle {
 }
 
 ///
-/// A progressively subdivided shape which can record
-/// the indices of the points and list out the individual
-/// triangles of the resulting shape.
+/// A subdivided shape generated from some [`BaseShape`] and a subdivision level.
 ///
-/// All base triangles specified by `S` in [`BaseShape`]
+/// The subdivided shape is defined, as is conventional in most 3D graphics systems,
+/// as a list of vertices, and a list of indices into the vertex list which connect
+/// the vertices into primitive shapes. [`Subdivided`] can provide triangle-list indices
+/// indices for solid surface rendering, and line-strip indices for wireframe rendering.
+///
+/// All main triangles specified by `S` in [`BaseShape`]
 /// are expected to be in counter clockwise winding.
```

Comment:
If I recall correctly, the library should technically work just fine with clockwise windings -- after all the triangle plane has no inherent winding. However, I never took the time to figure out the constants for a clockwise icosphere.

(You don't have to do anything about this, this is just a comment I wanted to mention)